builtin/buffer.mbt

```diff
@@ -28,6 +28,7 @@
 struct Buffer {
   mut bytes : Bytes
   mut len : Int
+  mut initial_capacity : Int
```

Comment:
can we keep initial_buffer instead, so that for small buffers it never allocates. 
It is also immutable